newm-server/src/main/kotlin/io/newm/server/features/marketplace/database/MarketplacePendingSaleEntity.kt

```diff
@@ -0,0 +1,29 @@
+package io.newm.server.features.marketplace.database
+
+import org.jetbrains.exposed.dao.UUIDEntity
+import org.jetbrains.exposed.dao.UUIDEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.lessEq
+import org.jetbrains.exposed.sql.deleteWhere
+import java.time.LocalDateTime
+import java.util.UUID
+
+class MarketplacePendingSaleEntity(id: EntityID<UUID>) : UUIDEntity(id) {
+    val createdAt: LocalDateTime by MarketplacePendingSaleTable.createdAt
```

Comment:
At some point, we should look at using uuidv7 ids. These have built-in timestamps so we don't need createdAt fields. That being said, I'm not sure how you could ever clean up previously existing data so maybe this isn't a great idea.